src/guides/v2.3/install-gde/continue-to-install.md

```diff
@@ -2,7 +2,7 @@
 group: installation-guide
 subgroup: Z_continue
 title:
-menu_title: Next&mdash;Installation part 2, installing
+menu_title: Next---Installation part 2, installing
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**